splitio/tests/test_redis_support.py

```diff
@@ -21,74 +21,6 @@
 from splitio.splits import (JSONFileSplitFetcher, SplitParser)
 from splitio.segments import JSONFileSegmentFetcher
 
-
-class RedisSegmentCacheTests(TestCase):
-    def setUp(self):
-        self.some_segment_name = mock.MagicMock()
-        self.some_segment_name_str = 'some_segment_name'
-        self.some_segment_keys = [mock.MagicMock(), mock.MagicMock()]
-        self.some_key = mock.MagicMock()
-        self.some_change_number = mock.MagicMock()
-        self.some_redis = mock.MagicMock()
-        self.a_segment_cache = RedisSegmentCache(self.some_redis)
-
-    def test_unregister_segment_removes_segment_name_to_register_segments_set(self):
-        """Test that unregister_segment removes segment name to registered segments set"""
-        self.a_segment_cache.unregister_segment(self.some_segment_name)
-        self.some_redis.srem.assert_called_once_with('SPLITIO.segments.registered',
-                                                     self.some_segment_name)
-
-    def test_get_registered_segments_returns_registered_segments_set_members(self):
-        """Test that get_registered_segments returns the registered segments sets members"""
-        self.assertEqual(self.some_redis.smembers.return_value,
-                         self.a_segment_cache.get_registered_segments())
-        self.some_redis.smembers.assert_called_once_with('SPLITIO.segments.registered')
-
-    def test_add_keys_to_segment_adds_keys_to_segment_set(self):
```

Comment:
i think these test should work, only registere/unregister segment operations are deprecated

Reply:
wrong assumption, I will add it again